python/tvm/autotvm/task/dispatcher.py

```diff
@@ -458,11 +468,16 @@ def __init__(self, records):
             Otherwise, it is an iterator.
         """
         # pylint: disable=import-outside-toplevel
-        from ..record import load_from_file
+        from ..record import load_from_file, load_from_buffer
 
         super(ApplyGraphBest, self).__init__()
-        if isinstance(records, str):
+        if isinstance(records, str, bytes, PathLike):
```

Comment:
i thought the types were supposed to be in a tuple, like:
```suggestion
        if isinstance(records, (str, bytes, PathLike)):
```